src/Hl7.Fhir.Support.Poco/Utility/FhirVersionUtility.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Hl7.Fhir.Specification;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Hl7.Fhir.Utility
+{
+    public class FhirReleaseParser
+    {
+        /// <summary>
+        /// Returns a FHIR version as an enum from a version number
+        /// </summary>
+        /// <param name="version">Fhir Release version number</param>
+        /// <returns>Official FHIR Release</returns>
+        public static FhirRelease Parse(string version)
+        {
+            return version switch
+            {
+                "0.01" => FhirRelease.DSTU1,
+                "0.05" => FhirRelease.DSTU1,
+                "0.06" => FhirRelease.DSTU1,
+                "0.11" => FhirRelease.DSTU1,
+                "0.0.80" => FhirRelease.DSTU1,
+                "0.0.81" => FhirRelease.DSTU1,
+                "0.0.82" => FhirRelease.DSTU1,
+
+                "0.4.0" => FhirRelease.DSTU2,
+                "0.5.0" => FhirRelease.DSTU2,
+                "1.0.0" => FhirRelease.DSTU2,
+                "1.0.1" => FhirRelease.DSTU2,
+                "1.0.2" => FhirRelease.DSTU2,
+
+                "1.1.0" => FhirRelease.STU3,
+                "1.4.0" => FhirRelease.STU3,
+                "1.6.0" => FhirRelease.STU3,
+                "1.8.0" => FhirRelease.STU3,
+                "3.0.0" => FhirRelease.STU3,
+                "3.0.1" => FhirRelease.STU3,
+                "3.0.2" => FhirRelease.STU3,
+
+                "3.2.0" => FhirRelease.R4,
+                "3.3.0" => FhirRelease.R4,
+                "3.5.0" => FhirRelease.R4,
+                "3.5a.0" => FhirRelease.R4,
+                "3.6.0" => FhirRelease.R4,
+                "4.0.0" => FhirRelease.R4,
+                "4.0.1" => FhirRelease.R4,
+
+                "4.2.0" => FhirRelease.R5,
+                "4.4.0" => FhirRelease.R5,
+                "4.5.0" => FhirRelease.R5,
+                "5.0.0" => FhirRelease.R5,
+                _ => throw new Exception($"Unknown FHIR version {version}")
+            };
+        }
+
+        /// <summary>
+        /// Returns the version number of the latest official FHIR releases
+        /// </summary>
+        /// <param name="fhirRelease">Official FHIR release</param>
+        /// <returns>Latest version number</returns>
+        public static string FhirVersionFromRelease(FhirRelease fhirRelease)
```

Comment:
I was wondering *what* we are using this for. I think I have seen it in Martijn's code, but this feels rather weird, since "the newest" version for a release is dynamic, this again would force us to push a new SDK version when there is a new FHIR patch release.  I am wondering whether Martijn uses this for creating a hard dependency or a dependency like R5 -> ">=4.5", which would give  us a bit more leeway. I'll take a look in his packaging library.

Reply:
I took at look, it's unclear, since the functions that use this FhirVersionFromRelease in the packaging library don't have references in the library itself. They are probably used by Simplifier.  Well, in that case, these functions are still ok, even if they don't return the very latest fhir version given a fhir release.

Reply:
How are we not going to forget that we need to keep this up-to-date?

Reply:
Maybe add this update step to the process here: https://github.com/FirelyTeam/firely-net-sdk/wiki/How-to-create-a-new-release